swarm.cabal

```diff
@@ -674,7 +674,7 @@ library swarm-tui
     swarm:swarm-scenario,
     swarm:swarm-util,
 
-  hs-source-dirs: src
+  hs-source-dirs: src/swarm-tui
```

Comment:
Add a `cabal-gild` line just above the `exposed-modules:` line, like so:
https://github.com/swarm-game/swarm/blob/d6c0fb006dc589fb8d95107c795fbe155e335f70/swarm.cabal#L545-L546

Then run the `scripts/normalize/cabal.sh` script, which will ensure that all of the files are in the expected directory.

Reply:
Fir this sublibrary, need to change `discover src/swarm-doc` to `discover src/swarm-tui`.

Reply:
Good catch, done!